content/vendor/guides/airgapped-existing-cluster.md

```diff
@@ -0,0 +1,523 @@
+```
+
+
+Now, let's ssh into the instance and bootstrap a minimal single-primary kubernetes cluster (details here:  https://kurl.sh/1010f0a  )
+
+```shell script
+gcloud compute ssh ${AIRGAP_CLUSTER} -- 'curl  https://k8s.kurl.sh/1010f0a  | sudo bash'
+```
+
+When that completes, we can deploy a minimal registry and verify it's running
+
+```shell script
+gcloud compute ssh ${AIRGAP_CLUSTER} -- \
+  'kubectl --kubeconfig ./admin.conf apply -f https://gist.githubusercontent.com/dexhorthy/7a3e6eb119d2d90ff7033a78151c3be2/raw/6c67f95367988d1a016635e3da689e2d998d458c/plain-registry.yaml'
+```
+
+This gist configures a basic auth htpasswd that configures a username/password for `kots/kots`, which we'll use later
+
+```shell script
+gcloud compute ssh ${AIRGAP_CLUSTER} -- \
+  'kubectl --kubeconfig ./admin.conf get pod,svc -n registry'
+```
+
+Now that the registry is up, let's verify that we can docker push/pull to it. We'll use the public IP attached to the instance.
+
+```shell script
+export CLUSTER_PUBLIC_IP=$( \
+  gcloud compute instances describe ${AIRGAP_CLUSTER} \
+  --format='get(networkInterfaces[0].accessConfigs[0].natIP)')
+docker login --username kots --password kots ${CLUSTER_PUBLIC_IP}:32000
+docker pull busybox
+docker tag busybox ${CLUSTER_PUBLIC_IP}:32000/busybox
+docker push ${CLUSTER_PUBLIC_IP}:32000/busybox
+```
+
+You may need to also add an `insecure-registy` entry to your workstation to allow pushing/pulling via http instead of https. If you're testing using docker-for-mac, you can add this via the setttings:
+
+![insecure registry](/images/guides/kots/airgap-existing-dfm-insecure-registry.png)
+
+
+Next, remove the machine's public IP. We'll use the kubeconfig from this server later.
+
+```shell script
+gcloud compute instances delete-access-config ${AIRGAP_CLUSTER}
+```
+
+Verify that internet access was disabled by ssh'ing via the jump box and trying to curl kubernetes.io. We'll forward the agent so that we can ssh the airgapped cluster without moving keys around
+
+```shell script
+gcloud compute ssh --ssh-flag=-A ${AIRGAP_JUMP} -- \
+  "ssh ${AIRGAP_CLUSTER} 'curl -v https://kubernetes.io'"
+```
+
+This command should hang, and you might see something with `Network is unreachable`:
+
+```text
+  0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0*   Trying 2607:f8b0:4001:c05::64...
+* TCP_NODELAY set
+* Immediate connect fail for 2607:f8b0:4001:c05::64: Network is unreachable
+  0     0    0     0    0     0      0      0 --:--:--  0:00:03 --:--:--     0
+```
+
+Once you're satisfied the instance has no outbound connectivity, you can Ctrl+C this command and we'll proceed to finalizing our workstation configuration.
+
+#### Final Workstation Setup
+
+Finally, let's verify our docker client on the workstation and make sure we have kubectl access properly configured before we do the full installation. We'll do this by ssh'ing to the workstation via the jump box.
```

Comment:
Is the purpose of the workstation to have an airgapped instance from where `kubectl kots install` command can be run? Is it just for convenience? If I had everything pulled onto the jumpbox and removed outbound internet access, the jumpbox could function as the workstation also right? Just trying to understand all the parts of the puzzle.